x-pack/filebeat/module/crowdstrike/_meta/docs.asciidoc

```diff
@@ -0,0 +1,79 @@
+[role="xpack"]
+
+:modulename: crowdstrike
+:has-dashboards: true
+
+== Crowdstrike module
+
+This is the filebeat module for the Crowdstrike Falcon using the Falcon https://www.crowdstrike.com/blog/tech-center/integrate-with-your-siem[SIEM Connector]. This module collects this data, converts it to ECS, and ingests it to view in the SIEM. By default, the Falcon SIEM connector outputs JSON formated Falcon Streaming API event data in JSON format.
+
+This module segments events forwarded by the Falcon SIEM connector into two datasets for endpoint data and Falcon platform audit data.
```

Comment:
```suggestion
```

Reply:
I was going to keep this since we're still doing this (two datasets), just from the same fileset. That okay?

Reply:
Oh, that's interesting. I didn't notice that the module was setting the `event.dataset` values. We've never had a case AFAIK where a fileset produced an `event.dataset` value other than the default of `{module}.{fileset}`. I don't think this will cause any issues. I think it would be good to callout the event.dataset values that it produces in this paragraph.